run/types/testing.ts

```diff
@@ -304,8 +307,10 @@ export type AccessibilityId =
   | '1 week'
   | '2 weeks'
   | 'Off'
+  | `${DMTimeOption} - Radio`
```

Comment:
I cannot link to it from here, but you can replace all the lines of from 298 to 309 with `| DMTimeOption`
![image](https://github.com/user-attachments/assets/54dfe74f-7096-4885-8035-431ee1f8f5b9)

Reply:
done